test/suite/test_txn19.py

```diff
@@ -286,30 +287,24 @@ def test_corrupt_log(self):
         self.corrupt_log(newdir)
         #self.show_logs(newdir, 'after corruption')
         salvage_config = self.base_config + ',salvage=true'
-        errfile = 'list.err'
-        outfile = 'list.out'
         expect_fail = self.expect_recovery_failure()
 
-        # In cases of corruption, we cannot always call wiredtiger_open
-        # directly, because there may be a panic, and abort() is called
-        # in diagnostic mode which terminates the Python interpreter.
-        #
-        # Running any wt command externally to Python allows
-        # us to observe the failure or success safely.
-        # Use -R to force recover=on, which is the default for
-        # wiredtiger_open, (wt utilities normally have recover=error)
-        self.runWt(['-h', newdir, '-C', self.base_config, '-R', 'list'],
-            errfilename=errfile, outfilename=outfile, failure=expect_fail,
-            closeconn=False)
-
         if expect_fail:
-            self.check_file_contains(errfile,
-                'WT_TRY_SALVAGE: database corruption detected')
+            with self.expectedStdoutPattern('.'):
```

Comment:
If feasible, we should have something specific to match in the pattern.  You can match multiple outputs, like
```
with self.expectedStdoutPattern('(this|that)'):
```

Reply:
done.